Library/Homebrew/manpages/brew.1.md.erb

```diff
@@ -105,6 +105,12 @@ can take several different forms:
   * `HOMEBREW_BOTTLE_DOMAIN`:
     If set, instructs Homebrew to use the given URL as a download mirror for bottles.
 
+  * `HOMEBREW_ARTIFACT_DOMAIN`:
+    If set, instructs Homebrew to use the given URL as a download mirror for bottles and binaries.
+    
+    *Note:* provide repository name in URL like
+    `HOMEBREW_ARTIFACT_DOMAIN=https://{URL of Artifact Manager}/{Repository name}`
```

Comment:
Should this not just be `HOMEBREW_ARTIFACT_DOMAIN=https://$URL_OF_ARTIFACT_MANAGER`? What would you set the repository name to?